File: Examples/VisualBasic/ProgrammersGuide/Working-with-Print/BySpecificPrinter.vb
Imports VisualBasic
Imports Aspose.Diagram
Imports System

Public Class BySpecificPrinter
    Public Shared Sub Run()
        'ExStart:BySpecificPrinter
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_Print()

        ' load source Visio diagram
        Dim diagram As New Diagram(dataDir & Convert.ToString("Drawing1.vsdx"))

        ' call the print method to print whole Diagram using the printer name
        diagram.Print("LaserJet1100")
        'ExEnd:BySpecificPrinter
    End Sub
End Class

File: Examples/CSharp/Working-with-Print/BySpecificPrinter.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Diagram.Examples.CSharp.Working_with_Print
{
    public class BySpecificPrinter
    {
        public static void Run()
        {
            // ExStart:BySpecificPrinter
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Print();

            // Load source Visio diagram
            Diagram diagram = new Diagram(dataDir + "Drawing1.vsdx");

            // Call the print method to print whole Diagram using the printer name
            diagram.Print("LaserJet1100");
            // ExEnd:BySpecificPrinter
        }
    }
}
